bugbug/model.py

```diff
@@ -286,6 +286,8 @@ def items_gen(self, classes):
                 if commit["node"] in classes
             )
 
+            bugzilla.download_bugs(all_bug_ids)
```

Comment:
We need to move this somewhere else, model.py is not the right place.

We'll probably have to move it in the bug_retriever script.

Reply:
@marco-c if i add this to (get_all_bug_ids) in labels.py , we don't need to call it, bug_retriever script will automatically take care of it, right?

Reply:
Yes